flepimop/gempyor_pkg/src/gempyor/testing.py

```diff
@@ -275,25 +275,26 @@ def sample_fits_distribution(
         return bool(np.greater(sample, 0.0))
 
 
-def sample_script(directory: Path, executable: bool) -> Path:
+def sample_script(name: str, directory: Path, executable: bool) -> Path:
```

Comment:
maybe make name default to "example" and make last argument, to preserve previous behavior?

Reply:
Addressed in 20dcfb94a270619885585c898e2811d73e4926f9